system/jhtree/keybuild.cpp

```diff
@@ -119,6 +114,7 @@ class CKeyBuilder : public CInterfaceOf<IKeyBuilder>
     CRC32EndHT crcEndPosTable;
     CRC32 headCRC;
     bool doCrc = false;
+    KeyBuildContext ctx;
```

Comment:
Is this used?

Reply:
no, historic location prior to refactoring